### DISCUSSION TITLE
"Namibia" BRouter Profil

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#386] [Show and tell]
I'm planning to travel to Namibia/South Africa in January.
My attempts with various cruiser profiles weren't really optimal.
Huge detours are taken to avoid gravel roads, or we drive through residential areas to avoid a coarse gravel main road....
That's why I made a halfway working BRouter profile.
I'll put it here in case anyone can use it.
I think it works well in regions with very few roads, many of which are gravel.

[Namibia.brf.txt](https://github.com/devemux86/cruiser/files/13464653/Namibia.brf.txt)

**Comment by devemux86** [comment]
@Leeb64 thanks for sharing the BRouter profile!

Did you modified the [car-vario](https://github.com/abrensch/brouter/blob/master/misc/profiles2/car-vario.brf) BRouter profile?

**Reply by Leeb64** [reply]
I modified the car (efficient) profile.
The rolling resistance increased from 232 to 8000 (shorter (gravel) routes are chosen) and in line 325 of the profile the value increased from 32 to 50 (avoid residential areas) maximum speed 80km/h.
